package.json

```diff
@@ -5,8 +5,9 @@
   "main": "lib/single-spa.js",
   "scripts": {
     "build": "webpack -p",
-    "build:dev": "webpack",
-    "watch": "webpack -w",
+    "build:dev": "webpack --config webpack.config.dev.js",
+    "build:analyze": "webpack --config webpack.config.analyze.js",
```

Comment:
new script for analyzing bundle size -- notice how small node_modules is now

<img width="1440" alt="screen shot 2018-04-18 at 9 42 09 pm" src="https://user-images.githubusercontent.com/5524384/38970217-685e7e5a-4351-11e8-8fac-940870355757.png">